invenio_vocabularies/contrib/subjects/schema.py

```diff
@@ -1,42 +1,22 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright (C) 2020-2021 CERN.
+# Copyright (C) 2021 Northwestern University.
 #
 # Invenio-Vocabularies is free software; you can redistribute it and/or
 # modify it under the terms of the MIT License; see LICENSE file for more
 # details.
 
 """Subjects schema."""
 
-from marshmallow import EXCLUDE, INCLUDE, Schema, fields, validate
-from marshmallow_utils.fields import Links
+from invenio_records_resources.services.records.schema import BaseRecordSchema
+from marshmallow import fields
 
+from invenio_vocabularies.services.schema import i18n_strings
 
-class MetadataSchema(Schema):
-    """Basic metadata schema class."""
 
-    class Meta:
-        """Meta class to accept unknown fields."""
+class SubjectSchema(BaseRecordSchema):
```

Comment:
even if not all the fields are used, we should inherit from `BaseVocabularySchema`. Mainly to keep consistency with the RFC 🤷‍♂️

Reply:
This still applies. It was agreed we should comply with the RFC when doing affiliations. We can discuss this with Lars on Monday, but I would go for the RFC since is what has been agree for the rest. If we decide to change, we need to change Affiliations too.